DREAM/tasdreamExternalTests.cpp

```diff
@@ -566,6 +566,41 @@ bool DreamExternalTester::performTests(TypeDREAMTest test){
 void testDebug(){
     cout << "Debug Test" << endl;
     cout << "Put here testing code and call this with ./dreamtest debug" << endl;
+
```

Comment:
If the code is ready, move this to a separate test or remove altogether.

Note if this is complete.

Reply:
I've removed this code and will set up a separate PR for testing.

The code should be complete and ready for merging now.